trlx/orchestrator/ppo_orchestrator.py

```diff
@@ -178,74 +200,98 @@ def make_experience(self, num_rollouts: int = 1024, iter_count: int = 0):  # noq
                         ref_logits = ref_logits.to(device)
 
             if self.trainer.config.model.model_arch_type == "seq2seq":
-                logprobs = logprobs_from_logits(
-                    logits[:, :-1, :], response_tensors[:, 1:]
-                )
-                ref_logprobs = logprobs_from_logits(
-                    ref_logits[:, :-1, :], response_tensors[:, 1:]
+                logprobs = logprobs_of_labels(logits[:, :-1, :], padded_samples[:, 1:])
+                ref_logprobs = logprobs_of_labels(
+                    ref_logits[:, :-1, :], padded_samples[:, 1:]
                 )
             else:
-                logprobs = logprobs_from_logits(logits, all_tokens)
-                ref_logprobs = logprobs_from_logits(ref_logits, all_tokens)
+                logprobs = logprobs_of_labels(logits, all_tokens)
+                ref_logprobs = logprobs_of_labels(ref_logits, all_tokens)
 
-            n = samples.shape[0]
+            n_samples: int = samples.shape[0]
             logprobs = logprobs.cpu()
             ref_logprobs = ref_logprobs.cpu()
             query_tensors = query_tensors.cpu()
```

Comment:
Might also rename `query` to `prompt` for consistency

Reply:
Done